src/test/java/by/xCaptin/collections/arraylist/MyArrayListTest.java

```diff
@@ -0,0 +1,192 @@
+package by.xCaptin.collections.arraylist;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.*;
+
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+public class MyArrayListTest {
+
+    static String FILEPATH = "C:\\Users\\User\\IdeaProjects\\AstonTaskCollection\\src\\test\\java\\by\\xcaptin\\collections\\values";
```

Comment:
```suggestion
private static String FILEPATH = "src/test/java/by/xCaptin/collections/values";
```
A лучше values положить в resources